### ISSUE TITLE
Bevy new version is out

### ISSUE THREAD

**Issue by marko-lazic** [sdfgeoff/blender_bevy_toolkit#49]
Hey everyone,

Bevy got updated from 0.6.0 to 0.7.0 sometime a while ago. Bevy Rapier also got updated from 0.12.1 to 0.13.2

I tried migrating this project myself. The most notable changes are in rapier_physics.rs and some in blender_mesh.rs. Unfortunately, I couldn't make my project run afterward.

We should migrate the project so that at least it runs new versions of Bevy and Rapier.

**Comment by YourRentIsDue** [comment]
Yep, I noticed this too. Since I've recently started work on a Bevy project, I'm going to attempt to migrate it, we'll see how that goes.

**Comment by YourRentIsDue** [comment]
I have managed to get my project compiling with Blender_Bevy_Toolkit after I edited it to run on the most recent version of Bevy and Rapier. I will check if the features still work and submit a pull request if they do!

**Comment by YourRentIsDue** [comment]
The pull request is submitted, waiting on tests from sdfgeoff, but if you're really itching to use this project now, my fork should be compatible with the most recent version of bevy and rapier. I have tested gravity and collisions, both seem to work fine.

**Comment by marko-lazic** [comment]
Well done. I was actually struggling with the new method of inserting rigid bodies into entities part. I couldn't find any migration guide and I was not using Rapier directly before.

**Comment by Type1J** [comment]
If anybody came to this issue trying to get blender_bevy_toolkit to compile in their project, there's an upstream dependency issue for the version of rapier3d used that you'll need to fix in your own (not blender_bevy_toolkit's) `Cargo.lock` file. Once you generate `Cargo.lock` by attempting to build once, if you have errors, then you take the entry for rapier3d (not bevy_rapier3d) from blender_bevy_toolkit's `Cargo.lock` and replace your `Cargo.lock`'s entry with it to fix them.
